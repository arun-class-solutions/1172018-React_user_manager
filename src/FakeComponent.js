import React, { Component } from "react";

class FakeComponent extends Component {
  render() {
    return (
      <div>Fake Component Updated</div>
    );
  }
}

export default FakeComponent;
