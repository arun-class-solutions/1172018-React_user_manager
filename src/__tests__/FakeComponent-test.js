import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import FakeComponent from "../FakeComponent";

configure({ adapter: new Adapter() });

// Test suite for UserList
describe("FakeComponent Test", () => {

  // Spec to check Enzyme snapshot against virtual DOM render of component
  it("Matches generated snapshot", () => {
    expect(shallow(<FakeComponent />)).toMatchSnapshot();
  });

});
