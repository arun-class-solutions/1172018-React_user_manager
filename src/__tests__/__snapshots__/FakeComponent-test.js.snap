// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FakeComponent Test Matches generated snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FakeComponent />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Fake Component Updated",
    },
    "ref": null,
    "rendered": "Fake Component Updated",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Fake Component Updated",
      },
      "ref": null,
      "rendered": "Fake Component Updated",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
